src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from '../Todo';
+
+export const TodoList = ({ todos }) => (
+  <>
+    <ul>
+      {todos.map(todo => (
+        <li
+          key={todo.id}
+          className={(todo.completed)
+            ? 'todo completed '
+            : 'todo not_complited'}
+        >
+          <Todo todo={todo} />
```

Comment:
```suggestion
          <Todo {...todo} />
```
use spread operator to pass props